OldTests/src/test/java/com/tle/webtests/test/workflow/rejection/NewVersionTest.java

```diff
@@ -15,81 +16,109 @@
 		wizard = SearchPage.searchAndView(context, itemFullName).adminTab().newVersion();
-		wizard.editbox(1, itemFullName2);
-		wizard.save().submit();
-
-		// check that there is a version that is live, and one that is
-		// moderating.
-		ItemAdminPage adminPage = new ItemAdminPage(context).load();
-		adminPage.exactQuery(itemFullName);
-		adminPage.setSort("datemodified");
-		ItemListPage itemList = adminPage.results();
+		saveItem(wizard, 1, itemFullName2);
+		// check that there is a version that is live, and one that is moderating.
+		ItemListPage itemList = getItemList(itemFullName);
 		assertEquals(itemList.getResultForTitle(itemFullName, 1).getStatus(), "live");
 		assertEquals(itemList.getResultForTitle(itemFullName2, 1).getStatus(), "moderating");
 		logout();
 
-		// Login as SimpleModerator and moderate the item at the first step.
-		logon("SimpleModerator", "``````");
-
-		taskListPage = new TaskListPage(context).load();
-		modResults = taskListPage.exactQuery(itemFullName2);
-		assertEquals(modResults.getStepName(itemFullName2), "moderation step 1");
-		modResults.moderate(itemFullName2).accept();
-
-		// After moderation, check that the item has gone life after the first
-		// moderation step
-		adminPage = new ItemAdminPage(context).load();
-		adminPage.exactQuery(itemFullName2);
-		adminPage.setSort("datemodified");
-		itemList = adminPage.results();
+		// Login as SimpleModerator and moderate the new version of this item at the first step.
+		logon(FIRST_MODERATOR_USERNAME, BACKTICK_PASSWORD);
+		doModeration(FIRST_MODERATOR_USERNAME, BACKTICK_PASSWORD, STEP_NAME_ONE, itemFullName2,false);
+		// After moderation, check that the item has gone life after the first moderation step
+		itemList = getItemList(itemFullName2);
 		assertEquals(itemList.getResultForTitle(itemFullName2, 1).getStatus(), "live");
 		logout();
 
 		// Login as the SecondStepModerator and check that the item is still
 		// there to moderate, and is still live once moderated from second step.
-		logon("SecondStepModerator", "``````");
-		taskListPage = new TaskListPage(context).load();
-		modResults = taskListPage.exactQuery(itemFullName2);
-		ModerationView tasksTab = modResults.moderate(itemFullName2);
-		tasksTab.assignToMe();
-		tasksTab.accept();
+		logon(SECOND_MODERATOR_USERNAME, BACKTICK_PASSWORD);
+		doModeration(SECOND_MODERATOR_USERNAME,BACKTICK_PASSWORD,"", itemFullName2, true);
 		logout();
 	}
 
+	private void doModeration(String username, String password, String stepName, String itemFullName, Boolean lastModeration) {
+		TaskListPage taskListPage = new TaskListPage(context).load();
+		ModerateListSearchResults modResults = taskListPage.exactQuery(itemFullName);
+		if(!lastModeration)
+		{
+			assertEquals(modResults.getStepName(itemFullName), stepName);
+			modResults.moderate(itemFullName).accept();
+		}
+		else
+		{
+			ModerationView tasksTab = modResults.moderate(itemFullName);
+			tasksTab.assignToMe();
+			tasksTab.accept();
+		}
```

Comment:
For this if/else the braces should follow the statement.

Shame we don't have formatting here yet, but we will once we merge this repo in.